pkg/gui/controllers/helpers/app_status_helper.go

```diff
@@ -93,3 +107,36 @@ func (self *AppStatusHelper) renderAppStatus() {
 		}
 	})
 }
+
+func (self *AppStatusHelper) renderAppStatusSync(stop chan struct{}) {
+	go func() {
+		ticker := time.NewTicker(time.Millisecond * 50)
+		defer ticker.Stop()
+
+		// Forcing a re-layout and redraw after we added the waiting status;
+		// this is needed in case the gui.showBottomLine config is set to false,
+		// to make sure the bottom line appears. It's also useful for redrawing
+		// once after each of several consecutive keypresses, e.g. pressing
+		// ctrl-j to move a commit down several steps.
+		_ = self.c.GocuiGui().ForceLayoutAndRedraw()
+
+		self.modeHelper.SetSuppressRebasingMode(true)
+		defer func() { self.modeHelper.SetSuppressRebasingMode(false) }()
+
+	outer:
+		for {
+			select {
+			case <-ticker.C:
+				appStatus := self.statusMgr().GetStatusString()
+				self.c.SetViewContent(self.c.Views().AppStatus, appStatus)
+				viewsToRedraw := []*gocui.View{
+					self.c.Views().AppStatus, self.c.Views().Options, self.c.Views().Information,
```

Comment:
Here we're redrawing all the views in the bottom line

Reply:
Renamed the variable and added a comment in e58d797fe6e7.